Getting Started instructions on open-liberty README.md don't work

**Describe the bug**
Running the `Getting Started` instructions on the [open-liberty](https://github.com/OpenLiberty/open-liberty) README.md returns the following error:

```
Josephs-MBP:sample-getting-started josephcass$ mvn clean package liberty:run-server
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.174 s
[INFO] Finished at: 2020-07-08T14:55:29+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Could not find goal 'run-server' in plugin io.openliberty.tools:liberty-maven-plugin:3.1 among available goals undeploy, install-feature, dev, test-stop, dump, display-url, java-dump, debug, start, run, status, create, clean, stop, compile-jsp, test-start, uninstall-feature, install-server, package, deploy, configure-arquillian -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoNotFoundException
```

**Steps to Reproduce**
Take the following steps: 
```
git clone https://github.com/OpenLiberty/sample-getting-started.git
cd sample-getting-started
mvn clean package liberty:run-server
```

**Expected behavior**
The liberty server should start successfully

**Diagnostic information:**
 - OpenLiberty Version: Open Liberty 20.0.0.7
 - Java Version: 
```
openjdk version "13.0.2" 2020-01-14
OpenJDK Runtime Environment AdoptOpenJDK (build 13.0.2+8)
Eclipse OpenJ9 VM AdoptOpenJDK (build openj9-0.18.0, JRE 13 Mac OS X amd64-64-Bit 20200118_155 (JIT enabled, AOT enabled)
OpenJ9   - 6968c18d7
OMR      - 7a1b0239a
JCL      - 9c5a41bd5f based on jdk-13.0.2+8)
```
- Maven version:
```
Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T20:00:29+01:00)
Maven home: /usr/local/Cellar/maven/3.6.1/libexec
Java version: 13.0.2, vendor: AdoptOpenJDK, runtime: /Library/Java/JavaVirtualMachines/adoptopenjdk-13-openj9.jdk/Contents/Home
Default locale: en_GB, platform encoding: UTF-8
OS name: "mac os x", version: "10.15.5", arch: "x86_64", family: "mac"
```
 - server.xml configuration: https://github.com/OpenLiberty/sample-getting-started/blob/master/pom.xml
Importantly: liberty-maven-plugin version = 3.1

**Additional context**
The fix should be as simple as updating `mvn clean package liberty:run-server` to `mvn clean package liberty:run`

--- a/README.md
+++ b/README.md
@@ -32,7 +32,7 @@ A lightweight open framework for building fast and efficient cloud-native Java m
 ### Launch a sample app using [Maven](https://maven.apache.org/)
     git clone https://github.com/OpenLiberty/sample-getting-started.git
     cd sample-getting-started
-    mvn clean package liberty:run-server
+    mvn clean package liberty:run
 
 Open browser to http://localhost:9080/ and explore the demo application.
 

